content/applications/hr/fleet/service.rst

```diff
@@ -86,61 +296,223 @@ clicked, sort the data in different ways.
-default :guilabel:`Date` column, which sorts the information in chronological order (January to
-December) instead of alphabetical order.
+To add a service record from the list view, click the :guilabel:`New` button, and a service form
+loads. :ref:`Enter all the information <service/service-form>` on the service form.
+
+The form automatically saves as data is entered.
 
 Kanban view
 ~~~~~~~~~~~
 
-To view services by their stage, click on the :guilabel:`Kanban icon`, which is the second icon in
-the top right, and appears as four black squares in a cube.
+To view services by their stage, click the :guilabel:`Kanban` button, which is the second icon in
+the top-right corner, and appears as two different length bars beneath a line.
+
+All services are organized by service type, and appear in the corresponding Kanban column.
+
+The number of repairs for each type of service appears in the far-right of each Kanban column
+header.
+
+The collective status of the scheduled activities for each service type appears in the color-coded
+bar beneath each Kanban column title. Repairs with activities scheduled in the future appear green,
+activities due today appear yellow, overdue activities appear red, and repairs with no activities
+scheduled appear gray.
+
+Each Kanban card displays a color-coded activity-related icon, such as a :guilabel:`🕘 (clock)` icon
+or :guilabel:`📞 (phone)` icon, for example. These icons indicate both the type of scheduled
+activity and the status. The status of the activity corresponds to the colors in the status bar.
+Click on an activity icon to view the details of the specific activity.
+
+The length of the color bar is proportionate to the number of corresponding activities with that
+specific status in that particular stage. Hover over a color section to reveal the number of service
+records in that specific grouping.
+
+.. image:: service/kanban.png
+   :align: center
+   :alt: The service records presented in a Kanban view.
+
+To view **only** the records with a specific status, click the desired color bar section. The
+background color for the column changes to a pale hue of the same color (either green, yellow, red,
+or gray), and the color bar appears striped instead of solid. **Only** repairs and services with the
+selected status appear in the column.
+
+.. image:: service/kanban-status.png
+   :align: center
+   :alt: The oil change service records showing only repairs with past-due activities.
+
+.. _service/schedule-activity:
+
+Schedule activities
+*******************
+
+To schedule an activity for a repair or service from the Kanban view, click the activity icon in the
+lower-right corner of the service record, and click :guilabel:`+ Schedule an activity`. A
+:guilabel:`Schedule Activity` pop-up window appears.
+
+.. note::
+   Depending on what, if any, kind of activity is scheduled, the activity icon may appear
+   differently, such as a :guilabel:`⏰ (clock)`.
```

Comment:
the ending of this sentence is a bit confusing... can you clean-up/clarify it a touch during your revision? thanks